<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOverWindow</class>
 <widget class="QWidget" name="GameOverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameOverWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="lableText1">
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>The Winner Is : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWinnerPlayerId">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>13</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>player* </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelText2">
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Score : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWinnerScore">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>13</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>****</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelWinner">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>14</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Winner Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWinnerName">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>11</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Input your name here for ranking.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NameConfirmBtn">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 黑体 Std R</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
	border: 2px solid rgb(170, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QListWidget" name="rankingList_name">
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>test</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>b</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>c</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>d</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>e</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>f</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>g</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>i</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="rankingList_score">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1234</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelText3">
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Best player ever 👇</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelText4">
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Their Score 👇</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="BackToMenuBtn">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border: none;background-color: rgb(255, 255, 255);}QPushButton:hover{border: 2px solid rgb(170, 255, 255);}</string>
          </property>
          <property name="text">
           <string>Back To Menu</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="RetryBtn">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
	border: 2px solid rgb(170, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>Retry</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="ExitBtn">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 黑体 Std R</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
	border: 2px solid rgb(255, 255, 0);
}</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
